docs/user/wireguard.md

```diff
@@ -125,30 +165,31 @@ Now start wg0 to verify the configuration is OK:
 sudo wg-quick up wg0
 ```
 
-Example output:
+::: details Example output
 ```
-pi@manyberry:~$ sudo wg-quick up wg0
+pi@raspberrypi:~$ sudo wg-quick up wg0
 [#] ip link add wg0 type wireguard
 [#] wg setconf wg0 /dev/fd/63
 [#] ip -4 address add 10.0.0.1/32 dev wg0
 [#] ip link set mtu 1420 up dev wg0
 [#] iptables -A FORWARD -i wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o wlan0 -j MASQUERADE
 ```
+:::
 
-We also want Wireguard to start when the Pi starts:
+We also want Wireguard to start when the server starts:
 ```bash
 sudo systemctl enable wg-quick@wg0
 ```
 
-Now reboot your Pi for the system changes to take effect:
+Now reboot your server for the system changes to take effect:
```

Comment:
We have to make a choice whether the guide to be for a general server, or the Pi. I have no problem in writing it for the Pi, and letting the users sort out how to do it for a general case.